tests/test_loan_transitions.py

```diff
@@ -238,3 +239,97 @@ def test_checkout_on_unavailable_item(loan_created, db, params,
         loan = current_circulation.circulation.trigger(
             loan_created, **dict(params)
         )
+
```

Comment:
Can we split it in more but less complicated tests?

Reply:
I updated the test with some comments to follow the test case scenario.